kyo-prelude/shared/src/main/scala/kyo/kernel/package.scala

```diff
@@ -2,26 +2,6 @@ package kyo
 
 package object kernel:
 
-    /** Identity type constructor.
```

Comment:
I've moved these type helpers to the `kyo` package since `Aspect` declarations can now use them as well.